apps/masterbots.ai/lib/hooks/use-profile.tsx

```
@@ -0,0 +1,60 @@
+'use client'
+
+import * as React from 'react'
+import { getUserByUsername, updateUserPersonality } from '@/services/hasura'
+import { useSession } from 'next-auth/react'
+
+interface profileContextProps {
+ getuserInfo: (username: string) => Promise<any>
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)